.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 270
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 90
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "status: accepted"
+  - "status: verify"
```

Comment:
I think 1 year of not verifying if an issue is indeed an issue is plenty. I have been applying this label to issues which I have not been able to reproduce (due to time constraint or they just did not happen when I tried reproducing). Pls correct me if the label means something else
```suggestion
```

Reply:
I agree this label does not belong this scope. I added it because when @prlanzarin and I were closing some issues a couple of months ago we used it to mark some old issues that needed verification from their authors (other BigBlueButton members). I suggest we run with it a couple of weeks until we are sure about them.